### PR TITLE
add Support for VST3 plug-ins

### PR DESCRIPTION
**Pull request by mofeing**
### Changes to the core
- [x] Add new stanza 'vst3_plugin'
- [x] Add new default directory/location for 'vst3_plugin': ~/Library/Audio/Plug-Ins/VST3
- [x] Add new command argument: --vst3-plugindir=PATH
- [x] Update doc for this command in [all_stanzas.md](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/all_stanzas.md), [readme.md](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/readme.md), [brew-cask.1.md](https://github.com/caskroom/homebrew-cask/blob/master/doc/man_page/brew-cask.1.md) and [brew-cask.1](https://github.com/caskroom/homebrew-cask/blob/master/man/man1/brew-cask.1) .
- [x] Tested locally.

### CODE REVIEW

File: doc/cask_language_reference/all_stanzas.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ Each Cask must declare one or more *artifacts* (i.e. something to install).
 | `service`          | yes                           | relative path to a Service that should be linked into the `~/Library/Services` folder on installation
 | `audio_unit_plugin`| yes                           | relative path to an Audio Unit plugin that should be linked into the `~/Library/Audio/Components` folder on installation
 | `vst_plugin`       | yes                           | relative path to a VST Plugin that should be linked into the `~/Library/Audio/VST` folder on installation
+| `vst3_plugin`       | yes                           | relative path to a VST3 Plugin that should be linked into the `~/Library/Audio/VST3` folder on installation
```
Comment (vitorgalvao) [inline doc/cask_language_reference/all_stanzas.md]: One space too many. Misaligned.

**Comment by vitorgalvao** [comment]
Any @caskroom/maintainers opposed to this?

**Comment by adidalal** [comment]
Did a quick look through, LGTM

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 9d8d16c611eae9e1a15d09016f401fd1fe6ca870. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by mofeing** [comment]
You're welcome.

**Pull request closed without merging**